src/main/kotlin/no/nav/amt/distribusjon/journalforing/JournalforingService.kt

```diff
@@ -86,20 +85,39 @@ class JournalforingService(
             endring = false,
         )
 
-        journalforingstatusRepository.insert(
+        journalforingstatusRepository.upsert(
             Journalforingstatus(
                 hendelseId = hendelseId,
                 journalpostId = journalpostId,
+                skalJournalfores = true,
             ),
         )
 
         log.info("Journalførte hovedvedtak for deltaker ${deltaker.id}")
     }
 
     private fun handleEndringsvedtak(hendelse: Hendelse) {
+        journalforingstatusRepository.upsert(
+            Journalforingstatus(
+                hendelseId = hendelse.id,
+                journalpostId = null,
+                skalJournalfores = true,
+            ),
+        )
         log.info("Endringsvedtak for hendelse ${hendelse.id} er lagret og plukkes opp av asynkron jobb")
     }
 
+    private fun handleEndringSomIkkeJournalfores(hendelse: Hendelse) {
```

Comment:
Trenger vi egentlig å lagre noe her?

Jeg innser også at tanken min rundt å ha en boolean kanskje er unødvendig. Man kunne bare lagret de som skal journalføres uten en `journalpostId` også hentet ut de hvor den er null. Hvis vi skal ha den så tenker jeg at `skalJournalføres` betyr at den ikke er journalført enda men skal bli det i fremtiden. 🤔

Reply:
Ja, det går kanskje an? Altså at vi oppdaterer spørringen her https://github.com/navikt/amt-distribusjon/blob/main/src/main/kotlin/no/nav/amt/distribusjon/hendelse/HendelseRepository.kt#L43 til å hente de som har et innslag der journalpostid er null? Og så dropper vi å lagre noe annet enn selve hendelsen for de som ikke skal journalføres? Det blir jo kanskje en forenkling ja.

Reply:
Vi vet iallefall at de som har en journalpostId er journalført, og de som mangler den ikke er journalført, så hvis vi bare lar være å lagre noe annet enn hendelsen for de som ikke skal journalføres så er nok det det enkleste ja :)